go.mod

```diff
@@ -259,3 +259,5 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
+
+replace github.com/flanksource/duty => ../duty
```

Comment:
@yashmehrotra TODO

Update after https://github.com/flanksource/duty/pull/223 is merged